Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14696,6 +14696,29 @@ end
 						Notes = "Logs the {{cCompositeChat}}'s human-readable text into the server console and logfile using 'warning' severity (red text)",
 					},
 				},
+				LOGRAW =
+				{
+					{
+						Params =
+						{
+							{
+								Name = "Message",
+								Type = "string",
+							},
+						},
+						Notes = "Logs a text into the server console and logfile using 'warning' severity (red text)",
```

Comment:
Is this right? Looks like severity is just regular below not warning.

Reply:
I thought I changed this :( I saw that too

Reply:
Oh yeah: I changed it below but not there